e2e/Readme.md

```diff
@@ -5,7 +5,7 @@ e2e tests for the Civo Cloud Controller Manager.
 These tests can be run:
 
 ```bash
-CIVO_API_KEY=.... go test --timeout 30 -v ./e2e/...
+CIVO_API_KEY=.... go test -timeout 20m -v ./...
```

Comment:
>[!NOTE]
The command was incorrect, so I have fixed it. Additionally, since it didn't finish within 30 seconds, I have changed the timeout to 20 minutes.